Dockerfile

```diff
@@ -1,3 +1,14 @@
 FROM 137112412989.dkr.ecr.us-west-2.amazonaws.com/amazonlinux:2017.03.1.20170812@sha256:8b2b0d94a896c5f100ee60ad274bc7677e0038bc450642e6d142f503bb8ab9b8
```

Comment:
If this is the image going into dockerhub and is called `aws-lambda-haskell-platform` it should probably live in a separate repo.

Reply:
Let's leave it here for now. I'm going to open an RFC (internally) about where it should live.